Pod/Classes/VOKManagedObjectMapper.h

```diff
@@ -6,15 +6,16 @@
 #import <Foundation/Foundation.h>
 #import <CoreData/CoreData.h>
 
+#import "VOKCoreDataCollectionTypes.h"
 #import "VOKNullabilityFeatures.h"
 
 #import "VOKManagedObjectMap.h"
 
 NS_ASSUME_NONNULL_BEGIN
 
-typedef void(^VOKPostImportBlock)(NSDictionary *inputDict, NSManagedObject *outputObject);
+typedef void(^VOKPostImportBlock)(VOKStringToObjectDictionary *inputDict, NSManagedObject *outputObject);
 
-typedef void(^VOKPostExportBlock)(NSMutableDictionary *outputDict, NSManagedObject *inputObject);
+typedef void(^VOKPostExportBlock)(VOKStringToObjectMutableDictionary *outputDict, NSManagedObject *inputObject);
```

Comment:
Not returning, but passing—because the point of the post-export block is to be able to modify the output dictionary, I think.